### ISSUE TITLE
Additional columns

### ISSUE THREAD

**Issue by kalons7** [fman-community/fman#184]
At least Version, Attributes, Extension.  Right-click column header to select columns to display.

**Comment by kek91** [comment]
Related to #48 and #67 (those are more specific, perhaps fitting as sub-tasks)

**Comment by ababak** [comment]
Maybe even allow making it a custom Python-scripted column?

**Comment by mherrmann** [comment]
@ababak That's the plan!

**Comment by oskretc** [comment]
I'll buy coffee for everyone who votes for this feature :stuck_out_tongue_winking_eye:

**Comment by mherrmann** [comment]
:laughing: @oskretc the old poll-rigger ;)

**Comment by mherrmann** [comment]
But more seriously - which columns do you want @oskretc?

**Comment by o314** [comment]
And i'll offer a mug for an open API too :bowtie:

**Comment by mherrmann** [comment]
The API [is open](https://fman.io/docs/api#Column)(?) The only thing that's missing is #294. Or did I miss something?

**Comment by o314** [comment]
-spoken too quickly-

**Comment by o314** [comment]
Oops sorry Michael.

I have not remarked this point yet. 
I will investigate it as soon as i can.

**Comment by karstengresch** [comment]
@mherrmann I'm in need of a 'Added' column (macOS differs between 'Date Created', 'Date Modified' and 'Date Added'). When dealing w/ larger parallel file operations, it gets difficult to find the latest files as their created/modified dates can differ from the added dates.

**Comment by oskretc** [comment]
file extension column will be nice to have...  many times I sort by type and delete/move a bunch of files

**Comment by jamdev** [comment]
I agree that sorting by file extension is an important feature and will enable a lot of use cases.

**Comment by mherrmann** [comment]
@jamdev please express that the issue is important for you by leaving a thumbs-up :+1: on the issue description. This updates the "vote count" and lets me take your wish into account more easily.

**Comment by jamdev** [comment]
Thanks @mherrmann - I added my thumbs up to #294 - hopefully you'll aggregate the two :)

**Comment by mherrmann** [comment]
I'll do my best :-)

**Comment by camxcube** [comment]
I really need a Date Added column.

**Comment by djquartz** [comment]
I found this from a Google search for how to add columns. I'm looking for the Music Genre column to be added.
